pom.xml

```diff
@@ -282,6 +282,8 @@
               <version>3.8.0</version>
               <configuration>
                 <release>${jdk.version}</release>
+                <source>${jdk.version}</source>
+                <target>${jdk.version}</target>
```

Comment:
`-release N` means `-source N -target N -bootclasspath N` - so this should be unnecessary unless you are using JDK8?

Reply:
i thought so too but for some reason using adoptopenjdk-11 with intellij it defaults source version to 7

Reply:
Could be an IntelliJ bug (making this change the pragmatic thing to do)

Reply:
I think it's the issue of intellij. intellij doesn't translate this, so it will mess up the IDE.

Reply:
yeah but for now i dont think there's any better solutions than what i am doing here. should we just merge this one first?